app/controllers/application_controller.rb

```diff
@@ -4,6 +4,16 @@ class ApplicationController < ActionController::Base
   before_action :set_locale
   before_action :set_search
 
+  helper_method :current_order
+
+  def current_order
+    if !session[:order_id].nil?
```

Comment:
```
@order = Order.where(user_id: current_user.id, status: 'processing').find_or_create
```
=> Tìm order với điều kiện là current_user và status là processing, nếu không có thì tạo tương ứng. 
Toàn bộ cái If của em viết thành thế này là đc, đọc thử lại về hàm đi, không sử dụng session. Em thử test lại xem

Reply:
@tuanhv-0694 da vang de chieu toi em doc a